### PR TITLE
Order GET /sessions by descending start date

### PR DESCRIPTION
**Pull request by brittanylau**
### Notion ticket link

[Order GET /sessions by descending start date](https://www.notion.so/uwblueprintexecs/Developer-Hub-30e9b6c82af24054b31acf7c5e822c89?p=f8608b78745b412499e05bcfd55c8da2)

### Implementation description

- updated the queryset on the `SessionViewSet` to order sessions by descending start date, with null start dates last
- updated test cases

### Steps to test

1. `python manage.py test`

### What should reviewers focus on?

- squeaky clean code 🧼 

### Checklist

- [x] My PR name is descriptive and in imperative tense
- [x] I have run the linter
- [x] I have requested a review from the PL, and/or other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: enrolments/views.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+from django.db.models import F
```
Comment (7tint) [inline enrolments/views.py:1]: F 😔

File: enrolments/views.py [hunk 2]
```diff
@@ -16,7 +17,7 @@ class SessionViewSet(
     mixins.ListModelMixin,
     mixins.RetrieveModelMixin,
 ):
-    queryset = Session.objects.all()
+    queryset = Session.objects.all().order_by(F("start_date").desc(nulls_last=True))
```
Comment (7tint) [inline enrolments/views.py:20]: wow didn't know this! very cool 😋

**Review by 7tint — APPROVED** [review]
tests work, cool sorting code :)

File: enrolments/views.py (hunk 1 above)
Comment (brittanylau) [reply enrolments/views.py:1]: F 😔

**Pull request merged**